content/docs/contributing/README.md

```diff
@@ -60,9 +60,16 @@ before the meeting. The meeting chair will try to get to everything that was on
 We try to record these meetings and put them on YouTube so they can be checked later – if you don't want to appear on video please keep
 your camera off!
 
+[comment]: <> (Changing this heading will break the reference link at https://cert-manager.io/docs/contributing/#slack, so avoid modifying it.)
```

Comment:
Adding comments in these docs is a bit weird; there's a guide here: https://github.com/cert-manager/website/?tab=readme-ov-file#task-adding-a-comment-in-documentation